inclusive-design/wecount.inclusivedesign.ca
# feat: enable role-based permissions for Netlify CMS (fix #470)

## Description

Returns 404 for non-admin users when visiting `/admin/`.

## Steps to test

1. Ensure that your user has the `admin` role in the Netlify Identity dashboard.
1. Log in via the log in link on the home page (at the bottom).
2. Visit `/admin/`.
3. Log out.
4. From the Netlify Identity dashboard, remove the `admin` role from your user.
5. Log in again.
6. Visit `/admin/`.

**Expected behavior:**

1. With the `admin` role, `/admin` returns the Content Management System.
2. Without the `admin` role, `/admin` returns a 404 error.

## Additional information

Not applicable.

## Related issues

Not applicable.
